src/main/webapp/registration.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Registration Page</title>
+    <link rel="stylesheet" href="registration.css">
+    <script src="registration.js"></script>
+  </head>
+  <body>
+      <form action "/data" method="POST">
+            <p>Are you a student or a tutor?</p>
+            <input type="radio" name="role" value="student"> Student<br>
+            <input type="radio" name="role" value="tutor"> Tutor<br><br>
```

Comment:
Can someone select both of these?

Reply:
Since both of their names are set to "role" it deselects the other if you try to select them both

Reply:
Cool! It might be worth thinking how to modify this later when we allow a user to both be a student and a tutor at the same time.